clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -42,6 +34,14 @@
 import org.apache.kafka.common.quota.ClientQuotaFilter;
 import org.apache.kafka.common.requests.LeaveGroupResponse;
 
+import java.time.Duration;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
```

Comment:
nit: is there a reason we moved these imports?

Reply:
This is invoked by the IDE spotless plugin, but I think this is reasonable since we are going to add import order checkstyle here:https://github.com/apache/kafka/pull/10428/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R615, and java.* will be put last.